openai-core/src/commonMain/kotlin/com.aallam.openai.api/chat/FunctionCall.kt

```diff
@@ -0,0 +1,18 @@
+package com.aallam.openai.api.chat
+
+import com.aallam.openai.api.BetaOpenAI
+import kotlinx.serialization.Serializable
+import kotlin.jvm.JvmInline
+
+/**
+ * The function to use for generating the completion.
+ */
+@JvmInline
+@Serializable
+@BetaOpenAI
+public value class FunctionCall(public val functionCall: String) {
```

Comment:
Indeed, it appears that this can also be an [object](https://github.com/openai/openai-openapi/blob/8be079fc413316349aedce5d6630e55180b001d9/openapi.yaml#L2403-L2414)!

In that case, we might consider defining `FunctionCall` as a sealed class or interface, with two implementations for instance - `FunctionCallString` and `FunctionCallObject`.

Reply:
Done